src/Microsoft.OData.Edm/Vocabularies/Annotations/IEdmVocabularyAnnotation.cs

```diff
@@ -30,5 +30,11 @@ public interface IEdmVocabularyAnnotation : IEdmElement
         /// Gets the expression producing the value of the annotation.
         /// </summary>
         IEdmExpression Value { get; }
+
+        /// <summary>
+        /// Gets whether the annotation uses a default value
+        /// (not defined with a provided value).
+        /// </summary>
+        bool UsesDefault { get; }

```

Comment:
as per separate comment, changing a public interface is a breaking change (it breaks any apps that may have implemented the interface, since they won't have implemented this property) so we have to define  new interface that derives from IEdmVocabularyAnnotation and adds the UsesDefault method.

Reply:
Done ✅